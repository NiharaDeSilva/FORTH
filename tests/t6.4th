10 20 + 2 ^ 10 / 5 * .
20 5 + 10 * 25 / 4 ^ CR .
12 2 ^ 6 - 60 + 5 * CR .
