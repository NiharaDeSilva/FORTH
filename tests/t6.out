450
10000
990